src/__tests__/actions/actions.test.js

```diff
@@ -0,0 +1,111 @@
+import * as actions from '../../actions';
```

Comment:
I discovered that holding tests in same directory as real source code can looks messy especially with larger project but
have advantages of managing dependencies.
I'm not a best person to decide about this but in my feeling we should either configure webpack for resolve dependencies relativelly to project directories:

```javascript
{
  resolve: {
    alias: {
      reducers: "./src/reducers"
    }
  }
}
```

or hold test files side by side with tested files

```
reducers/users.js
reducers/users.spec.js
``` 

With larger and larger project this should tend to webpack solution but for sure it can't be `'../../..'` because it's simply unreadable for humans

Reply:
Got it. I also thought about keeping them together with tested cfilede. Then maybe it would need a better structure for the source code files (e.g. group them additionally not to end up with hundreds of files in one directory).

I also thought about doing some reading up on having it set up in similar fashion as Rails and specs. Thanks for the suggestions.